### PR TITLE
Add CMMI Logo

### PR DESCRIPTION
**Pull request by thallaren**
Added CMMI Logo png to \img\partnerships and changed _content/work/04_partnerships.md to include it.

### CODE REVIEW

**Review by thallaren — COMMENTED** [review]
Adding old stsiinc.com news page to pluribus site to test LOE of adding old links to reduce 404s

File: content/news/2016/05/stsi-to-sponsor-devopsdaysdc.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (bengm) [inline content/news/2016/05/stsi-to-sponsor-devopsdaysdc.md:1]: I'm not sure it helps us to recreate 4 year old news, unless we're getting lots of hits at that URL.

Comment (thallaren) [reply content/news/2016/05/stsi-to-sponsor-devopsdaysdc.md:1]: We aren't getting lots of hits there at all. Reducing 404s was the only reason if it is deemed a bad "look", sure not a priority in my book.

**Comment by thallaren** [comment]
closing to remove old news article

**Pull request closed without merging**